scripts/eval/eval.py

```diff
@@ -20,21 +22,52 @@
                                        build_tokenizer)
 
 
-def load_model(model_cfg, tokenizer, num_retries):
+def load_model(model_cfg, tokenizer, fsdp_config, num_retries):
+    # Restrict model init_device to 'meta' and 'cpu',
+    # using 'cuda' vs. 'cuda:id' is tricky and can lead to common user errors
+    # when multiple GPUs are available.
+    # Also 'meta' is only valid when using FSDP
+    init_context = contextlib.nullcontext()
```

Comment:
Is this section an exact clone of the one for train? If so, can we make it a util in foundry proper and import?

Reply:
Yup, done.